docs/codegen-tour.md

```diff
@@ -687,6 +688,26 @@ fancy `RecordData` struct that would be a thin wrapper around a record plus the
 parent data, and which would implement the record getters, but deref to the
 record otherwise.... I'm really not sure.
 
+### <a id="arrays"></a> arrays
+
+Arrays of scalars, offsets, and records occur frequently in the spec. How we
+represent these arrays in codegen depends on what we know about the size of the
+contents the array. There are essentially three cases:
+
+- if the contents of an array have a fixed uniform size, known at compile time, then we
+  represent the array as a rust slice: `&[T]`. This is true for all scalars
+  (including offsets) as well as the majority of records.
```

Comment:
"majority of" is less helpful than being specific, e.g. 

- Use `&[T]` if the array is made up of fixed size entries and the size is known at compile time
   - The size of all scalars, including offsets, is fixed and known at compile time
   - Records made up of scalars or records that are ultimately made up of scalars